File: PRAK505/PRAK505_2310817110011_Ghani-Mudzakir.c
#include <stdio.h>

void Biodata(int tahunlahir, char namalengkap[], char asalkota[])
{
    int tahunSekarang = 2023;
    int umur = tahunSekarang-tahunlahir;
    printf("\nPerkenalkan Nama Saya : %s\n", namalengkap);
    printf("Umur Saya : %d\n", umur);
    printf("Saya adalah Angkatan : %d\n", tahunSekarang);
    printf("Asal Saya Dari : %s\n", asalkota);

}
int main() 
{
    int tahunLahir;
    char namalengkap[100], asalkota[100];
    scanf(" %d",&tahunLahir);
    scanf(" %[^\n]%*c",&namalengkap); 
    scanf(" %[^\n]%*c", &asalkota);
    Biodata(tahunLahir, namalengkap, asalkota);
    return 0;
}

File: PRAK505/PRAK505_2310817110011_Ghani-Mudzakir.py
def Biodata(tahunlahir,namalengkap,asalkota):
    tahunlahir = int(tahunlahir)
    tahunSekarang = 2023
    umur = tahunSekarang-tahunlahir
    print(f"\nPerkenalkan Nama Saya : {namalengkap}")
    print(f"Umur Saya : {umur}")
    print(f"Saya adalah Angkatan : {tahunSekarang}")
    print(f"Asal Saya Dari : {asalkota}")


tahunLahir = input() 
namaLengkap = input() 
asalKota= input()
tahunLahir = int(tahunLahir)
Biodata(tahunLahir, namaLengkap, asalKota)


